Wikipedia/Code/SavedViewController.swift

```diff
@@ -154,7 +154,9 @@ class SavedViewController: ViewController {
         navigationBar.addExtendedNavigationBarView(extendedNavBarView)
         navigationBar.addUnderNavigationBarView(underBarView)
         navigationBar.isBackVisible = false
-        
+
+        wmf_add(childController:savedProgressViewController, andConstrainToEdgesOfContainerView: progressContainerView)
```

Comment:
Diving into why exactly this works if it's before `currentView` but not after. Should be ok to roll with this fix in the mean time.